web/modules/custom/collection/collection.module

```diff
@@ -415,12 +415,15 @@ function collection_rdf_entity_view_alter(array &$build, EntityInterface $entity
  *   Amount of members.
  */
 function collection_member_count(RdfInterface $entity) {
-  return \Drupal::entityQuery('og_membership')
+  $query = \Drupal::database()->select('og_membership', 'om');
+  $query->leftJoin('users_field_data', 'ufd', 'om.uid = ufd.uid');
```

Comment:
Shouldn't we use an `innerJoin` here? If there is no user, the count should be gone too.